### PR TITLE
lout: Update to 3.42

### PR DESCRIPTION
**Pull request by judaew**
#### Description

- lout: switch to GitHub and update to 3.42
- lout: aligment only

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.1 21C52 x86_64
Xcode 13.2 13C90

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: textproc/lout/Portfile [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        william8000 lout 3.42
+github.tarball_from archive
+revision            0
+license             GPL-3+
+categories          textproc
+platforms           darwin freebsd
+maintainers         gmail.com:ggreif
```
Comment (pmetzger) [inline textproc/lout/Portfile:11]: If possible, we should get a github handle for the maintainer (who probably has no idea of this pull request's existence.)

Comment (judaew) [reply textproc/lout/Portfile:11]: Maybe it's Elias Pipping. I see his [commit in 2007](https://github.com/macports/macports-ports/commit/e534d63be18213ecf984b2c1579cbd522d968dd3#diff-a577ed9479e91af4a73e0525259550206cb1cce1312d2f54f7b113ce8ad89a69), when the port was created. I wrote to him at the indicated email.

Comment (ggreif) [reply textproc/lout/Portfile:11]: The old maintainer is me, happy to hand it over!

Comment (judaew) [reply textproc/lout/Portfile]: Good, thanks for the quick reply!

**Pull request merged**